Formula/pandoc.rb

```diff
@@ -22,11 +22,6 @@ class Pandoc < Formula
   depends_on "gmp"
```

Comment:
Alright, let's give it a shot.  If anything breaks we can revert.  Thanks for looking closer!